home/models.py

```diff
@@ -17,34 +19,36 @@ def __str__(self):
         return f"{self.name} | {self.email}"
 
     class Meta:
```

Comment:
Should maybe add an index on email if we expect to do queries on it to aggregate data...?

Reply:
Since the number of users are in the 100s, indexing won't really do anything significant. We can add that in if needed later